data/stats/get_flight_stats.py

```diff
@@ -0,0 +1,51 @@
+""" Gather flight statistics for each 
+    airport from given aiport flight data. 
+"""
+
+from dataclasses import dataclass, field
+from typing import TypedDict
+
+import pandas as pd
+
+
+class FlightDict(TypedDict):
+    aircraft_model: str | None
+    airline: str | None
+    destination_iata: str | None
+    destination_icao: str | None
+    flight_number: str
+    long_aircraft_name: str | None
+    origin_iata: str | None
+    origin_icao: str | None
+    scheduled_arrival: int
+    scheduled_departure: int
+
+
+@dataclass
+class FlightStats:
+    flights: pd.DataFrame
```

Comment:
As this is never used or accessed outside of the `__post_init__` would it make sense to make it an `InitVar`?